op5build/ninja.logrotate

```diff
@@ -0,0 +1,36 @@
+/var/log/op5/ninja.log {
+    su monitor apache
+    weekly
+    rotate 10
+    compress
+    dateext
+    dateformat -%Y%m%d
+    missingok
+    notifempty
+    postrotate
+        if command -v systemctl &>/dev/null; then
+            systemctl try-restart naemon > /dev/null 2>&1
+        else
+            /sbin/service naemon condrestart 2> /dev/null > /dev/null || true
+        fi
+    endscript
+    create 0664 monitor apache
+}
+/var/log/op5/ninja/* {
+    su monitor apache
+    weekly
+    rotate 10
+    compress
+    dateext
+    dateformat -%Y%m%d
+    missingok
+    notifempty
+    postrotate
+        if command -v systemctl &>/dev/null; then
+            systemctl try-restart monitor > /dev/null 2>&1
```

Comment:
monitor is the same as naemon, so the same applies here. I don't think (but please test) any restart would be needed at all, if anything it should be httpd.